torchreid/engine/builder.py

```diff
@@ -4,14 +4,15 @@
 
 from .image import (ImageAMSoftmaxEngine, MultilabelEngine, MultiheadEngine)
 
+NNCF_ENABLED_LOSS = ['softmax', 'am_softmax', 'am_binary']
 
 def build_engine(cfg, datamanager, model, optimizer, scheduler,
                  should_freeze_aux_models=False,
                  nncf_metainfo=None,
                  compression_ctrl=None,
                  initial_lr=None):
     if should_freeze_aux_models or nncf_metainfo:
-        if cfg.loss.name not in ['softmax', 'am_softmax', 'am_binary']:
+        if not any(loss_name in NNCF_ENABLED_LOSS for loss_name in cfg.loss.name.split(',')):
```

Comment:
Could you change from `any` to `all`? I think if not enabled loss comes, this if statement should raise error.